content/docs/release-notes/2019-19-12.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "release notes 2019-19-12"
+date: 2019-19-12
+releaseType:
+  - ui
+---
+
+### Email Content Testing for Dynamic Templates
+
+Test your email content against spam filters, validate hyperlinks, and preview HTML rendering across browsers and devices with Email Testing for Dynamic Templates. Monthly testing credits available for Trial, Pro and Premier plans. See our Email Testing [documentation](https://sendgrid.com/docs/ui/sending-email/email-testing/) for more details. 
```

Comment:
Monthly testing credits included for Trial, Pro and Premier plans, with additional credits available for purchase.